src/main/java/controllers/FeedController.java

```diff
@@ -60,7 +60,7 @@ private void runBrowseFeed() {
      * @param posts An ArrayList of Posts representing the selection of displayed Posts on a Feed.
      * @return a String representing a selection of displayed Posts' titles on a Feed.
      */
-    private String generateDisplayedPosts(ArrayList<Post> posts) {
+    public String generateDisplayedPosts(ArrayList<Post> posts) {
```

Comment:
Why do we need these to be public? If there is a good reason, ignore this.

Reply:
oh it's so that I can use the method in UserProfileController